examples/01_random_field/08_fourier.py

```diff
@@ -0,0 +1,38 @@
+"""
+Generating a Simple Periodic Random Field
+-----------------------------------------
+
+In this simple example we are going to learn how to generate periodic spatial
+random fields. The Fourier method comes naturally with the property of
+periodicity, so we'll use it to create the random field.
+"""
+
+import numpy as np
+
+import gstools as gs
+
+# We start off by defining the spatial grid.
+L = np.array((500, 500))
```

Comment:
Since L has the same values in each direction we could simplify this example by using a `size=500` variable and also pass this as a single period argument.

Reply:
Good idea, implemented.